.changeset/five-berries-rest.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
+---
+
+Remove constellation commands
```

Comment:
You're probably going to want to make this changeset more verbose, as it'll be displayed here: https://developers.cloudflare.com/changelog/?product=wrangler

Like maybe:

Remove `wrangler constellation` commands in favour of `wrangler ai` commands.